app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: "str", age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people.update({self.name: self})
+
+    def add_partner(self, pertner_name: str) -> None:
+        self.partner = Person.people[pertner_name]
```

Comment:
There is a typo in the variable name `pertner_name`. It should be `partner_name`.